Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ColorTypes"
 uuid = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
-version = "0.10.4"
+version = "0.10.5"
```

Comment:
I'm unclear about the ColorTypes roadmap. Is this version bump deliberately?

I ask this because this PR doesn't introduce any changes to the `src/` folder.

Reply:
It is intentional, but not definitive. It was just moved from PR #187.